files/bootstrap.sh

```diff
@@ -387,21 +396,28 @@ sed -i s,MASTER_ENDPOINT,$APISERVER_ENDPOINT,g /var/lib/kubelet/kubeconfig
 sed -i s,AWS_REGION,$AWS_DEFAULT_REGION,g /var/lib/kubelet/kubeconfig
 ### kubelet.service configuration
 
-if [[ "${IP_FAMILY}" == "ipv6" ]]; then
-      DNS_CLUSTER_IP=$(awk -F/ '{print $1}' <<< $SERVICE_IPV6_CIDR)a
-fi
+MAC=$(get_meta_data 'latest/meta-data/network/interfaces/macs/' | head -n 1 | cut -d'/' -f1 | sed 's/\/$//')
```

Comment:
Did you notice this break without the `cut -d'/' -f1` ?

Reply:
this is unfortunately a testing discrepancy with the `ec2-metadata-mock`, so this isn't strictly necessary when using the real IMDS but is when using the ec2-metadata-mock .

Reply:
submitted an issue to ec2-metadata-mock: https://github.com/aws/amazon-ec2-metadata-mock/issues/173

and one to add a metadata dump to a file: https://github.com/aws/amazon-ec2-metadata-mock/issues/174

Reply:
The issue is resolved and the cut has been removed! :D